lib/fastlane/plugin/wpmreleasetoolkit/actions/common/buildkite_pipeline_upload_action.rb

```diff
@@ -1,17 +1,19 @@
 module Fastlane
   module Actions
     class BuildkitePipelineUploadAction < Action
-      DEFAULT_ENV_FILE = File.join('.buildkite', 'shared-pipeline-vars').freeze
+      DEFAULT_BUILDKITE_PIPELINE_FOLDER = '.buildkite'.freeze
+      DEFAULT_ENV_FILE = File.join(DEFAULT_BUILDKITE_PIPELINE_FOLDER, 'shared-pipeline-vars').freeze
 
       def self.run(params)
-        pipeline_file = params[:pipeline_file]
+        pipeline_file = File.join(DEFAULT_BUILDKITE_PIPELINE_FOLDER, params[:pipeline_file])
```

Comment:
In case we want to maintain a tad of backwards compatibility or future flexibility, we might consider only doing that if `pipeline_file` is a relative path, not an absolute path?

```suggestion
        pipeline_file = params[:pipeline_file]
        pipeline_file = File.join(DEFAULT_BUILDKITE_PIPELINE_FOLDER, pipeline_file) unless File.absolute_path?(pipeline_file)
```

Reply:
That's a good idea 👌

Reply:
Remember to update the `ConfigItem`'s description to document this behavior wrt relative/absolute path 😉

Reply:
And also to add a unit test for it 😇